src/components/date-input/date-input.yaml

```diff
@@ -1,3 +1,57 @@
+params:
+- name: id
+  type: string
+  required: false
+  description: This is used for the main component and to compose id attribute for each item.
+- name: namePrefix
+  type: string
+  required: false
+  description: Optional prefix. This is used to prefix each `item.name` using `-`.
+- name: items
+  type: array
+  required: true
+  description: An array of input objects with name, value and classes.
+  params:
```

Comment:
This is missing `classes` – it wasn't in the README either 😢

Reply:
Done!